cnv/cnv_virtual_machines/cnv_importing_vms/cnv-importing-vmware-vm.adoc

```diff
@@ -12,6 +12,19 @@ If you import a VMware template, the wizard creates a virtual machine based on t
 include::modules/technology-preview.adoc[leveloffset=+1]
 :!FeatureName:
 
+.Prerequisites
+
+* You must create a VDDK image, push it to an image registry, and add it to the `v2v-vmware` ConfigMap.
+
+* There must be sufficient storage space for the imported disk.
```

Comment:
I think that this prereq *can* work in the assembly, but it also is directly relevant to the procedure module. I would keep it in the procedure, but I don't think it's *necessary* to move it back.